core/Tracker/TableLogAction.php

```diff
@@ -246,26 +246,25 @@ private static function normaliseActionString($actionType, $actionString)
     {
         $actionString = Common::unsanitizeInputValue($actionString);
 
-        if (self::isActionTypeStoredSanitized($actionType)) {
-            return Common::sanitizeInputValue($actionString);
+        if (self::isActionTypeStoredUnsanitized($actionType)) {
+            return $actionString;
         }
-        return $actionString;
+
+        return Common::sanitizeInputValue($actionString);
     }
 
     /**
      * @param $actionType
      * @return bool
      */
-    private static function isActionTypeStoredSanitized($actionType)
+    private static function isActionTypeStoredUnsanitized($actionType)
     {
         $actionsTypesStoredUnsanitized = array(
-            $actionType == Action::TYPE_PAGE_URL,
             $actionType == Action::TYPE_DOWNLOAD,
             $actionType == Action::TYPE_OUTLINK,
         );
 
-        $isStoredUnsanitized = in_array($actionType, $actionsTypesStoredUnsanitized);
-        return !$isStoredUnsanitized;
+        return in_array($actionType, $actionsTypesStoredUnsanitized);
```

Comment:
Thanks for the review. the logic is broken, I will take a look and maybe add some test